docs/make-docs.R

```diff
@@ -136,7 +136,7 @@ make_doc_hierarchy = function() {
     "pl", "Series", "DataFrame", "LazyFrame", "GroupBy",
     "LazyGroupBy", "ExprList", "ExprBin", "ExprCat", "ExprDT",
     "ExprMeta", "ExprName", "ExprStr", "ExprStruct",
-    "Expr", "IO", "RThreadHandle"
+    "Expr", "IO", "RThreadHandle", "SQLContext"
```

Comment:
"SQL" would be clearer

Reply:
I am not sure how this works, I assumed that SQLContext is the name of the actual structure used upstream and here it is related to the name of the Rd file.

Reply:
Yeah, I think the change should happen below actually, where we rename the Expr variants

Reply:
Not even that, forget about this comment, I'll take care of this later